### PR TITLE
Ops/split actions

### PR DESCRIPTION
**Pull request by andycarrell**
@theramis LMK what you think - if you wanna I'll remove the git on push config

### CODE REVIEW

File: .github/workflows/fetchy.yml [hunk 1]
```diff
@@ -30,7 +32,6 @@ jobs:
       - name: Run Fetchy
         run: npm run fetch-boi
       - uses: EndBug/add-and-commit@v7
-        if: github.event_name == 'workflow_dispatch' || github.event_name == 'repository_dispatch'
```
Comment (andycarrell) [inline .github/workflows/fetchy.yml:33]: We technically wouldn't need this 👋

File: .github/workflows/fetchy.yml [hunk 2]
```diff
@@ -1,10 +1,12 @@
 name: Run Fetchy Fetch Boi
 on:
-  pull_request:
   repository_dispatch:
   workflow_dispatch:
     branches:
       - main
+  push:
+    branches:
+      - ops/split-actions
```
Comment (andycarrell) [inline .github/workflows/fetchy.yml:9]: This is kinda how you can test it per PR / branch

File: .github/workflows/verify-fetchy.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        uses: actions/setup-node@v2.4.1
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+          cache: "npm"
+      - name: Install dependencies
+        run: npm ci
+      - name: Run Fetchy
+        run: npm run fetch-boi
+      - name: Print results
+        run: git ls-files --others --exclude-standard | xargs -I filename cat filename
```
Comment (andycarrell) [inline .github/workflows/verify-fetchy.yml:29]: There's probably a better way ;)

Comment (theramis) [reply .github/workflows/verify-fetchy.yml:29]: TBH this isn't getting printed out at the moment so we don't need this. 

But if you want you can just do `git status`. It should be enough to tell us a new file got generated.

Comment (andycarrell) [reply .github/workflows/verify-fetchy.yml:29]: Interesting yeh ofc - we could purge the data folder prior to run, but like you say, probably not necessary

File: .github/workflows/verify-fetchy.yml [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    name: Verify Run Fetchy
+    runs-on: ubuntu-latest
+    env:
+      # set this secret, you can get the value from this discord message https://discord.com/channels/658523382442360866/903771050385502239/904216952141152328
+      DISCORD_WEBHOOK: $ {{ secrets.DISCORD_WEBHOOK }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2.3.4
+        with:
+          token: ${{ secrets.PAT }}
```
Comment (theramis) [inline .github/workflows/verify-fetchy.yml:16]:
```suggestion
```

You need to delete these two lines :P

File: .github/workflows/verify-fetchy.yml (hunk 3 above)
Comment (theramis) [reply .github/workflows/verify-fetchy.yml:29]:
```suggestion
        run: git status
```

**Comment by theramis** [comment]
I think I personally still prefer having it all in one file but can see that it's to think about in two different workflows since there are less permutations. 

So happy to do this.

**Comment by andycarrell** [comment]
Thanks @theramis - this might be one of the things I feel strongly about haha 🙏

**Review by theramis — APPROVED** [review]

**Comment by theramis** [comment]
> Acccctually, there might be another way, to have it all in one file, but have a second step dependent on the first.
> 
> I can look into that tomorrow, but would rather get this in and PRs passing?

nah alg, this is fine :)

**Pull request merged**